{"name": "Проверка авторизации пользователя с неверным логином", "status": "passed", "description": "Проверка авторизации пользователя с неверным логином и получения ошибки 401", "start": 1722448428846, "stop": 1722448429082, "uuid": "64dad179-6e3a-4a1f-94f7-efe294d162af", "historyId": "163fbac4a50b86dea237f47a01dd012d", "testCaseId": "163fbac4a50b86dea237f47a01dd012d", "fullName": "tests.test_login_user.TestLoginUser#test_no_such_login", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login_user"}, {"name": "subSuite", "value": "TestLoginUser"}, {"name": "host", "value": "DESKTOP-MUFNET6"}, {"name": "thread", "value": "70688-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login_user"}]}